whois.go

```diff
@@ -169,21 +162,26 @@ func (c *Client) QueryAndParse(domain string) (*Response, error) {
 			default:
 				response.ExpirationDate, _ = time.Parse(time.RFC3339, strings.ToUpper(value))
 			}
-		} else if key == "paid-till" {
-			// example for ru/su domains -> paid-till: 2024-05-26T21:00:00Z
+		} else if key == "paid-till" || key == "state" || key == "status" {
 			if strings.HasSuffix(domain, ".ru") || strings.HasSuffix(domain, ".su") {
 				response.ExpirationDate, _ = time.Parse(time.RFC3339, strings.ToUpper(value))
 			}
-		} else if strings.Contains(key, "status") {
 			response.DomainStatuses = append(response.DomainStatuses, value)
-		} else if key == "state" {
-			// example for ru/su domains -> state: DELEGATED, VERIFIED
-			if strings.HasSuffix(domain, ".ru") || strings.HasSuffix(domain, ".su") {
-				response.DomainStatuses = strings.Split(value, ", ")
-			}
-		} else if strings.Contains(key, "name server") || strings.Contains(key, "nserver") {
+		} else if strings.Contains(key, "nserver") || strings.Contains(key, "name server") {
 			response.NameServers = append(response.NameServers, value)
 		}
 	}
+	if len(response.DomainStatuses) == 0 {
+		response.DomainStatuses = append(response.DomainStatuses, "No domain status found")
+	}
+	if len(response.NameServers) == 0 {
+		response.NameServers = append(response.NameServers, "No name servers found")
```

Comment:
Can you revert this? Some people may have written code that would use this value as a name server